README.md

```diff
@@ -57,11 +57,21 @@ It might be required when scanning from docker-daemon or cri-o to provide a user
 ### `extra-parameters`
 
 Additional parameters added to the secure-inline-scan container execution.
-
 ### `extra-docker-parameters`
 
 Additional parameters added to the `docker` command when executing the secure-inline-scan container execution.
 
+### `severity`
+
+Filter output annotations by severity. Default is "unknown".
+Possible values:
+- ciritcal
```

Comment:
typo: ` ciritcal` --> `critical` (here and in the source files)

Reply:
Fixed